doc/version_history.rst

```diff
@@ -6,6 +6,21 @@
 Version History
 ###############
 
+v0.6.0
+======
+
+| Switched to pre-commit
+| Switched to ts-conda-build
+| Switched to JenkinsShared for the conda build
+| Small code updates due to modifications in ts_xml for MTDome
```

Comment:
What does the `|` do? Please consider * instead of | for consistency.

I wonder if we can start dropping the Requires section from our version histories, if the dependencies are clear enough in another file (but which one?). Not sure we're there yet, but looking forward to it.

Reply:
Not sure where I got that from. I have changed them all to *.